rero_ils/modules/acq_invoices/jsonschemas/acq_invoices/acq_invoice-v0.0.1.json

```diff
@@ -0,0 +1,307 @@
+            }
+          }
+        },
+        "order_number": {
+          "title": "Order number",
+          "type": "string",
+          "minLength": 3
+        },
+        "quantity": {
+          "title": "Numbor of units",
+          "type": "integer",
+          "minimum": 1
+        },
+        "price": {
+          "title": "Price per unit",
+          "type": "number",
+          "minimum": 0
+        },
+        "discount": {
+          "title": "Discount amount",
+          "type": "number",
+          "minimum": 0
+        },
+        "total_price": {
+          "title": "Total price",
+          "type": "number",
+          "minimum": 0
+        }
+      }
+    }
+  },
+  "properties": {
+    "$schema": {
+      "title": "Schema of the resource",
+      "description": "Schema to validate Acquisition Invoice records against.",
+      "type": "string",
+      "minLength": 9,
+      "default": "https://ils.rero.ch/schema/acq_invoices/acq_invoice-v0.0.1.json",
+      "pattern": "^https://ils.rero.ch/schema/acq_invoices/acq_invoice-v([0-9]+?\\.){3}json$"
+    },
+    "pid": {
+      "title": "Acquisition Invoice ID",
+      "type": "string",
+      "minLength": 1
+    },
+    "invoice_number": {
+      "title": "Invoice number",
+      "type": "string",
+      "minLength": 1,
+      "form": {
+        "validation": {
+          "validators": {
+            "valueAlreadyExists": {
+              "term": "invoice_number"
+            }
+          },
+          "messages": {
+            "alreadyTakenMessage": "The invoice number is already taken"
+          }
+        }
+      }
+    },
+    "invoice_price": {
+      "title": "Invoice price",
+      "type": "number"
+    },
+    "invoice_status": {
+      "title": "Invoice status",
+      "type": "string",
+      "enum": [
+        "in progress",
+        "valid for payment",
+        "deleted"
+      ],
+      "default": "in progress",
+      "form": {
+        "options": [
+          {
+            "value": "in progress",
+            "label": "In progress"
+          },
+          {
+            "value": "valid for payment",
+            "label": "Valid for payment"
+          },
+          {
+            "value": "deleted",
+            "label": "Deleted"
+          }
+        ]
+      }
+    },
+    "invoice_date": {
+      "title": "Invoice date",
+      "type": "string",
+      "format": "date",
+      "pattern": "\\d{4}-((0[1-9])|(1[0-2]))-(((0[1-9])|[1-2][0-9])|(3[0-1]))$",
+      "validationMessage": "Should be in the following format: 2022-12-31 (YYYY-MM-DD).",
+      "form": {
+        "type": "datepicker",
+        "placeholder": "Choose a date",
+        "wrappers": [
+          "form-field"
+        ],
+        "validation": {
+          "messages": {
+            "pattern": "Should be in the following format: 2022-12-31 (YYYY-MM-DD)."
+          }
+        }
+      }
+    },
+    "notes": {
+      "title": "Notes",
+      "description": "Notes.",
```

Comment:
Remove descriptions is it is the same as title. Check other properties.

Reply:
ok